Automattic/jetpack
# push-to-mirrors: Consider missing mirror as an error

#### Changes proposed in this Pull Request:

This seems like the more useful behavior, but is technically a breaking change in the package, hence the major version bump.

In the monorepo here, this will not prevent any merges, but will result in us being pinged in Slack after every merge until the needed mirror gets created.

#### Other information:


#### Jetpack product discussion


p1669037520481709/1669025975.301269-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Create a build artifact, or download and unpack one from a recent build.
* Modify the build to include a mirror that does not exist (edit mirrors.txt and create/move the corresponding directory). Leave some valid mirrors in mirrors.txt after the invalid one.
* Run the `push-to-mirrors.sh` script locally, after setting `BUILD_BASE`, `GITHUB_ACTOR`, and `GITHUB_REF`. Be sure `CI` is **not** set in your environment so it doesn't try to actually push.
* Check that the script reported the error and still continued to process later pushes, and in the end exited with a non-zero exit code.
